applications/psd_pipeline/advanced_network_connectors/vita49_rx.cu

```diff
@@ -226,14 +226,11 @@ void Vita49ConnectorOpRx::compute(
         InputContext& op_input,
         OutputContext& op_output,
         ExecutionContext& context) {
-  auto burst_opt = op_input.receive<in_t>("in");
-  if (!burst_opt) {
-    return;
-  }
-  auto burst = burst_opt.value();
+  BurstParams *burst;
+  auto status = get_rx_burst(&burst, 0, CONTEXT_QUEUE_ID);
```

Comment:
we should call `address_to_port` here as well to retrieve the port id given an interface name, since we lost the holoscan output port connection

Reply:
They don't specify the interface name in their config and always assume 1 port, which is why I did it this way.

Reply:
but with dpdk we know the port could not be 0 depending on which one you use from your NIC, no?

Reply:
DPDK always starts at 0 regardless of which port you have in the config